app/models/blacklight/icon.rb

```diff
@@ -2,30 +2,43 @@
 
 module Blacklight
   class Icon
-    attr_reader :icon_name
+    attr_reader :icon_name, :aria_hidden, :label, :role
     ##
     # @param [String, Symbol] icon_name
     # @param [Hash] options
     # @param [String] classes additional classes separated by a string
-    def initialize(icon_name, classes: '', aria_hidden: false)
+    # @param [Boolean] aria_hidden include aria_hidden attribute
+    # @param [Boolean] label include <title> and aria-label as part of svg
+    # @param [String] role role attribute to be included in svg
+    def initialize(icon_name, classes: '', aria_hidden: false, label: true, role: 'image')
```

Comment:
could we just pass through e.g. `aria: { hidden: true }` and pass all those properties (whatever they may be) through? Or even `html_options: {}`?